source/mainnet/net/resources/legal.rst

```diff
@@ -9,10 +9,7 @@ The following topics relate to terms and conditions of use as well as notices an
 .. toctree::
    :maxdepth: 1
 
-   terms-and-conditions-dw
```

Comment:
I did keep them here but made them orphans. This means that the file does not have to appear in any TOC. I thought it would be better that way so they weren't visible.